auxiliary_files/config_files/mumble-server.logrotate.in

```diff
@@ -0,0 +1,10 @@
+/var/log/@MUMBLE_SERVER_BINARY_NAME@/mumble-server.log {
```

Comment:
The other instances of mumble-server can also be replaced using `MUMBLE_SERVER_BINARY_NAME`

Reply:
Is this log-file created by systemd or is this the log file that the server is supposed to create?

Reply:
To my knowledge that is what is supposed to be created by `mumble-server`.

Systemd services are supposed to use [systemd-journald](https://man.archlinux.org/man/core/systemd/systemd-journald.8.en), which does not concern itself with plaintext logs, but uses a binary file type. FWIW, running mumble-server in foreground (`-fg`) should automatically have it output its stdout/stderr to systemd-journald!

This means that logrotate config files are not necessary on any distributions using systemd.

Reply:
Okay, I see. In that case the path to the log-file seems to be non-deterministic as well. In either case though that means that the name of the log file is determined by what is hard-coded in the server's code, which is not (directly) dependent on the binary's name, which is why I did not replace that occurrence of `mumble-server` with `@MUMBLE_SERVER_BINARY_NAME@`.

However, given that when using systemd, this file is not needed anyway, I think we could even consider completely removing it. Afaik the vast majority of modern systems uses systemd anyway, so I see little reason to maintain corner-case init system files in our repo...